focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -670,9 +670,9 @@ class BrowserFragment :
         val requestDesktop = tab.content.desktopMode
 
         val addToHomescreenDialogFragment = AddToHomescreenDialogFragment.newInstance(
-            tab.content.url,
-            tab.content.titleOrDomain,
-            tab.trackingProtection.enabled,
+            url = tab.content.url,
+            title = tab.content.titleOrDomain,
+            blockingEnabled = !tab.trackingProtection.ignoredOnTrackingProtection,
```

Comment:
This change matches another similar usecase - https://searchfox.org/mozilla-mobile/rev/3efcbbbe98c6bde158c0ea6df9326aef211f6bf8/firefox-android/focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt#887 so LGTM.